### DISCUSSION TITLE
Widget Titles in Dark Mode have bad Contrast (v3.5)

### DISCUSSION THREAD

**Discussion by v0tti** [netbox-community/netbox#12426] [General]
Is it just me or are the widget titles of the new dashboard in v3.5 pretty unreadable in dark mode? Especially the default "white on light grey" titles are pretty bad. They seem fine in light mode but the color gets faded when changing to dark mode.

Maybe the contrast can be tweaked a little bit?

**Comment by abhi1693** [comment]
There is an issue for this https://github.com/netbox-community/netbox/issues/12219
